src/app/content/hooks/dynamicStyles.ts

```diff
@@ -10,7 +10,7 @@ const hookBody: ActionHookBody<typeof receiveBook> = (services) => async() => {
   const loadingBook = loadingBookSelector(state);
 
   if (loadingBook || !book || !book.style_href ||
-      !book.loadOptions.booksConfig.books[book.id]?.dynamicStyles) {
+      book.loadOptions.booksConfig.books[book.id]?.dynamicStyles === false) {
```

Comment:
There is a potential behavior change here if the book config is completely missing but book and book.style_href are present somehow, but I guess that fits with the intent of the PR.

Just a little surprising it does not complain about coverage due to the `?` here. I'm guessing there's an unrelated spec somewhere that covers this case but doesn't check if the dynamic styles are loaded.